io.github.jakubmelka.pdf4qt.json

```
@@ -0,0 +1,44 @@
+{
+    "id": "io.github.jakubmelka.pdf4qt",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "qbs",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DWITH_TESTS:BOOL=OFF"
+            ],
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.qt.io/official_releases/qbs/1.23.0/qbs-src-1.23.0.tar.gz",
+                    "sha256": "c86aa775446aec728bcbbed782ec128f4e6e2c26536710017343e684bb616d7a"
+                }
+            ]
+        },
+        {
+            "name": "PDF4QT",
+            "buildsytem": "simple",
```

And here is the typo:
```suggestion
            "buildsystem": "simple",
```